controllers/gamesC.js

```diff
@@ -49,23 +50,41 @@ exports.getComments = (req, res, next) => {
 };
 
 exports.postComment = (req, res, next) => {
-    if (
-        !req.body.username ||
-        !req.body.body ||
-        Object.keys(req.body).length !== 2
-      ) {
-        next({ status: 400, msg: "incorrectly formatted comment body" });
-        return;
-      }
-      if (req.params.review_id % 1 !== 0) {
-        next({ status: 400, msg: "invalid id" });
-        return;
-      }
-    insertComment([req.params.review_id, req.body.username, req.body.body])
-      .then((comment) => {
-        res.status(201).send({ comment });
-      })
-      .catch((err) => {
-        next(err);
-      });
-  };
+  if (
+    !req.body.username ||
+    !req.body.body ||
+    Object.keys(req.body).length !== 2
+  ) {
+    next({ status: 400, msg: "incorrectly formatted comment body" });
+    return;
```

Comment:
This error handling looks great, just needs a test!

Reply:
you've responded to part of my task 7 code here but i've reworked it for the current task 8- should be good once i submit the pull request again!